devices/flash-flexspi/fspi/fspi.c

```diff
@@ -86,11 +81,41 @@ __attribute__((section(".noxip"))) void flexspi_lutUpdate(flexspi_t *fspi, u32 i
 }
 
 
+__attribute__((section(".noxip"))) void flexspi_lutUpdateEntries(flexspi_t *fspi, u32 index, const u32 *lutTable[], size_t elems, size_t count)
+{
+	u32 lutCopy[64];
+	const u32 *src;
+	unsigned int i, j;
+
+	for (i = 0; i < elems; ++i) {
+		src = *(lutTable++);
```

Comment:
imo `src = lutTable[i]` is much more readable

Reply:
Done.